connector/src/main/scala/com/basho/riak/spark/rdd/ReadConf.scala

```diff
@@ -31,6 +31,7 @@ case class ReadConf (
   val fetchSize: Int = ReadConf.DefaultFetchSize,
   val splitCount: Int = ReadConf.DefaultSplitCount,
   val tsTimestampBinding: TsTimestampBindingType = ReadConf.DefaultTsTimestampBinding,
+  val tsFieldName: String = null,
```

Comment:
Let's rename this to something more specific than time series Field.. for example: timeRangeField